backend/src/main/java/com/google/sps/servlets/SearchMatchServlet.java

```diff
@@ -116,10 +118,11 @@ private boolean isExpired(Participant participant) {
   }
 
   /** Send JSON response for expired participant that has been removed from datastore */
-  private void sendExpiredResponse(HttpServletResponse response) throws IOException {
+  private void sendExpiredResponse(HttpServletResponse response, Participant participant)
```

Comment:
It looks like `participant` is not used in this function. Please remove it.

Reply:
Ack